src/packages/cli/src/args.ts

```diff
@@ -188,12 +194,9 @@ export default function (
         command = ["$0", flavor];
         defaultPort = 8545;
         break;
-      default:
```

Comment:
As far as I can tell, this code path is unreachable, as the only valid flavors are "ethereum" and "filecoin", which are handled above.